server/testutil/buildbuddy/buildbuddy.go

```diff
@@ -0,0 +1,173 @@
+package buildbuddy
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/phayes/freeport"
+
+	bazelgo "github.com/bazelbuild/rules_go/go/tools/bazel"
+)
+
+const (
+	readyCheckPollInterval = 500 * time.Millisecond
+	readyCheckTimeout      = 30 * time.Second
```

Comment:
Can you comment what these are / how someone might set them?

Reply:
Done